source/app-development/tutorials-interactive-apps.rst

```diff
@@ -0,0 +1,13 @@
+.. _app-development-tutorials-interactive-apps:
+
+Tutorials: Interactive Apps
+===========================
+
+A curated list of tutorials for developing interactive apps.
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Tutorials
+
+   tutorials-interactive-apps/add-jupyter
+   tutorials-interactive-apps/add-custom-queue
```

Comment:
I notice when I navigate between the subsections of add-custom-queue the navigation on the left disappears because on a new page you are scrolled to the top and you have to scroll down to see. The readthedocs theme I think supports "subprojects" https://docs.readthedocs.io/en/latest/faq.html#how-do-i-host-multiple-projects-on-one-cname (or that might just be the readthedocs site itself...). My guess is this would result in the left navigation being devoted entirely to that subproject, which could be "App Development" or even "Interactive App Development". This is outside the scope of this PR though, maybe worth opening another issue.

Reply:
Is this due to a smaller window? I am not seeing this on my window:

![image](https://user-images.githubusercontent.com/4260509/34627303-11e3160e-f22e-11e7-9b04-15673388e00a.png)

I can explore sub-projects, but like you mentioned it might be better to open a new issue for that (I can open that issue if you want, or you can). Not sure how time-consuming that exploration may be, and I know you want this out before the webinar.

Reply:
Yeah smaller window. Make be browser height small enough so "Use a Local Static List" in the left nav doesn't appear without you scrolling.

That said, absolutely, open a new issue, don't do it here, and lets not even think about touching that issue till Feb.